src/pages/dashboard.jsx

```diff
@@ -1,16 +1,31 @@
-import React from 'react'
+import React, {Fragment} from 'react'
+import Media from 'react-media'
 import { Dashboard } from '../components/Dashboard/Dashboard'
+import { DashboardMobile } from '../components/Dashboard/DashboardMobile'
 import { AddTransactionButton } from '../components/AddTransactionButton/AddTransactionButton'
 import { SelectMenuModal } from '../components/SelectMenuModal/SelectMenuModal'
 import { SelectStatisticsMenu } from '../components/SelectStatisticsMenu/SelectStatisticsMenu'
 
+
 export const DashBoard = () => {
   return (
-    <>
-    <Dashboard/>
+  <>
+    <Media queries={{
+      small: "(max-width: 768px)",
+      medium: "(min-width: 769px)"
+    }}>
+      {matches => (
+        <Fragment>
+          {matches.small && <DashboardMobile/>}
+          {matches.medium && <Dashboard/>}
```

Comment:
można to zrobić za pomocą tenary `matches.small ? <DashboardMobile/> : <Dashboard/>`